glottolog3/__init__.py

```diff
@@ -137,6 +137,14 @@ def main(global_config, **settings):
         views.languages,
         renderer='language/bpsearch_html.mako')
 
+    config.add_route_and_view(
+        'glottolog.add_identifier', 
+        '/resource/identifier/add',
```

Comment:
Let's use RESTful API design. This endpoint should be POST request to `/identifiers`. I believe you can also use `config.add_route('function_name', '/identifiers')`, which will just use function_name, which needs a `@view_config` modifier. You can search up this modifier to see how it's used in glottolog, or check out this resource: https://zaiste.net/building_a_restful_api_with_pyramid_resource_and_traversal/